app/controllers/idv/usps_controller.rb

```diff
@@ -58,15 +58,15 @@ def failure
       redirect_to idv_usps_url unless performed?
     end
 
-    def pii
+    def pii(pii_params)
       hash = {}
-      update_hash_with_address(hash)
+      update_hash_with_address(hash, pii_params)
       update_hash_with_non_address_pii(hash)
       hash
     end
 
-    def update_hash_with_address(hash)
-      profile_params.each { |key, value| hash[key] = value }
+    def update_hash_with_address(hash, pii_params)
+      pii_params.each { |key, value| hash[key] = value }
```

Comment:
Isn't this really just the same as?

```suggestion
      hash.merge!(pii_params)
```

Reply:
yep, and it's always a blank hash, so it probably makes sense to duplicate the original address PII hash rather than iterate copying one by one?

What do you think of 4bd6c2210 for combining this idea with suggestion in https://github.com/18F/identity-idp/pull/4498/files#r538834610?

Reply:
Almost! I ended up pushing what I meant in f3b08b7a66455780bb6b72a91cc0ef712c729d77 since I figured it would be easier to show